examples/px4-offboard-mode.cpp

```diff
@@ -649,7 +649,7 @@ static int setup_timeout()
 
 int main(int argc, char *argv[])
 {
-    char *ip = (char *)"127.0.0.1";
+    char *ip = strdup("127.0.0.1");
     int port = 5760;
 
     if (argc > 1) {
```

Comment:
This leak memory every time user provides the IP:port.
I will squash the fix and merge.

Reply:
thanks